Firmware/ultralcd.cpp

```diff
@@ -5342,7 +5332,7 @@ static void lcd_main_menu()
     MENU_ITEM_FUNCTION_P(PSTR("power panic"), uvlo_);
 #endif //TMC2130_DEBUG
 
-    if ( ( IS_SD_PRINTING || usb_timer.running() || (lcd_commands_type == LcdCommands::Layer1Cal)) && (current_position[Z_AXIS] < Z_HEIGHT_HIDE_LIVE_ADJUST_MENU) && !homing_flag && !mesh_bed_leveling_flag) {
+    if ( BABYSTEP_ALLOWED() ) {
```

Comment:
beware: the `current_position[Z_AXIS] < Z_HEIGHT_HIDE_LIVE_ADJUST_MENU` is important

Reply:
Github marks this as outdate due to me rebasing the branch to sync with MK3. This comment is still relevant 👍

Reply:
[e3b9e9e](https://github.com/prusa3d/Prusa-Firmware/pull/4063/commits/e3b9e9efe0aeb9361faa7a7e93c877f250c540fe) This is now applied everywhere so it's consistant in behavior.